app/controllers/infections_controller.rb

```diff
@@ -8,6 +8,7 @@ def create
     @infection = Infection.new(infection_params)
     @infection.organism = @organism
     @infection.user = current_user
+    @infection.status = "active"
```

Comment:
do we really need this, if the column already has a default value?

Reply:
@Kimon-Haars you are right, it was left from my initial testing. let me remove it.